ShopifySharp/Entities/FulfillmentOrder.cs

```diff
@@ -0,0 +1,78 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace ShopifySharp
+{
+    /// <summary>
+    /// An object representing a Shopify fulfillment order.
+    /// </summary>
+    public class FulfillmentOrder : ShopifyObject
+    {
+        /// <summary>
+        /// The unique numeric identifier for the shop.
+        /// </summary>
+        [JsonProperty("shop_id")]
+        public long? ShopId { get; set; }
+
+        /// <summary>
+        /// The unique numeric identifier for the order.
+        /// </summary>
+        [JsonProperty("order_id")]
+        public long? OrderId { get; set; }
+
+        /// <summary>
+        /// The unique numeric identifier for the location.
+        /// </summary>
+        [JsonProperty("assigned_location_id")]
+        public long? LocationId { get; set; }
+
+        /// <summary>
+        /// "mars-fulfillment".
+        /// </summary>
+        [JsonProperty("fulfillment_service_handle")]
+        public string FulfillmentServiceHandle { get; set; }
+
+        /// <summary>
+        /// "submitted".
+        /// </summary>
+        [JsonProperty("request_status")]
+        public string RequestStatus { get; set; }
+
+        /// <summary>
+        /// "open".
+        /// </summary>
+        [JsonProperty("status")]
+        public string Status { get; set; }
+
+        /// <summary>
+        /// Destination for the fulfillment.
+        /// </summary>
+        [JsonProperty("destination")]
+        public Address Destination { get; set; }
```

Comment:
Wondering if it might be better to create a new class `FulfillmentOrderDestination`.
Even if the `Address` class matches all fields, they might drift apart in the future...